src/components/skill/SkillHeading.astro

```diff
@@ -20,7 +20,7 @@ const { skillNumber, skillName, img } = Astro.props;
     </div>
   </div>
   <div
-    class="relative mx-auto max-w-7xl py-8 px-4 sm:py-16 md:y-32 sm:px-6 lg:px-8"
+    class="relative mx-auto max-w-7xl py-8 px-4 sm:py-16 md:py-32 sm:px-6 lg:px-8"

```

Comment:
This component is very similar to the Hero one. Can you check if we can avoid duplicates? For example, the waves could be an extra file.